util/site-dashboard/ot-nightly-results.js

```diff
@@ -41,27 +41,30 @@ const all_report_urls = [
   ["uart", ["https://reports.opentitan.org/hw/ip/uart/dv/latest/"]],
   ["usbdev", ["https://reports.opentitan.org/hw/ip/usbdev/dv/latest/"]],
   ["xbar", ["https://reports.opentitan.org/hw/top_earlgrey/ip/xbar_main/dv/autogen/latest/", "https://reports.opentitan.org/hw/top_earlgrey/ip/xbar_peri/dv/autogen/latest/"]],
-  ["chip", ["https://reports.opentitan.org/hw/top_earlgrey/dv/latest/"]],
   ["rv_core_ibex", ["https://ibex.reports.lowrisc.org/opentitan/latest/"]]
 ];
 
-async function fetch_block_report(block_report_url) {
+const chip_level_report_urls = [
+  ["chip_earlgrey", ["https://reports.opentitan.org/hw/top_earlgrey/dv/latest/"]],
+];
```

Comment:
TBH I think these results should be pre-combined rather than having the client fetching 20+ json files.

Reply:
FWIW, I think we fetch these reports in the [fetch_block_stats.py](https://github.com/lowRISC/opentitan/blob/master/util/site/fetch_block_stats.py) script and produce a single summary JSON file for clients to load.

Currently this script is run at site-build-time and the report is used for the interactive block diagram. I'd like this to run when the reports are regenerated instead, but we haven't got round to that.

Maybe these reports can use that summary file instead? Probably too much work to add to this PR, though.

Reply:
> TBH I think these results should be pre-combined rather than having the client fetching 20+ json files.

Indeed, this was built in a slight hurry along with a bunch of other doc stuff and no combined JSON was available. It'd make sense to produce one as part of the dvsim report rendering it does for regressions (which is what currently produces the individual JSONs). Though note the Ibex data will still need to come from the seperate JSON (as it's regression is separate to the OpenTitan nightly).

Reply:
> Maybe these reports can use that summary file instead? Probably too much work to add to this PR, though.

I'd prefer dvsim to produce the summary as part of the rest of the report rendering then everything to draw data from there. Reduces the number of moving parts involved.